en/docs/webhooks.md

```diff
@@ -2,3 +2,40 @@
 
 !!! info
     This section will describe Webhooks, its concepts, and its usage concerns/concepts in Choreo.
+
+## Introduction
+
+A webhook directs information from one application to another in real-time and triggers the receiving application to execute a specific process in response.
+
+A project can use a webhook to receive information from another project or an external application in order to execute a process. A project can also trigger another project or an external application by sending information to it via a webhook.
+
```

Comment:
We don't have the capability for a project to invoke other systems via a webhook. We will have it in the future. But, not yet.